packages/nodes-base/nodes/Postgres/v1/genericFunctions.ts

```diff
@@ -169,6 +170,7 @@ export async function pgQueryV2(
 	items: INodeExecutionData[],
 	continueOnFail: boolean,
 	overrideMode?: string,
+	resolveExpression?: boolean,
```

Comment:
Is there a case where we won't want to resolve expressions?

Reply:
Not right now. I guess I had backwards compatibility on my mind when adding this.